dask_sql/context.py

```diff
@@ -250,6 +250,13 @@ def create_table(
             **kwargs,
         )
 
+        if type(input_table) == str:
+            filepath = input_table
+            dc.filepath = input_table
```

Comment:
I like this. I feel like this should have always been an attribute in the DataContainer all along